### PR TITLE
DOC: Clarify votes to fulfill vacancies in bylaws

### PR DESCRIPTION
**Pull request by jhlegarreta**
Clarify bylaws concerning how votes are casted to fulfill vacancies.

### CODE REVIEW

**Review by thewtex — COMMENTED** [review]
Thanks for the proposal. Let's discuss at the next ISC meeting.

File: src/pages/bylaws.js [hunk 1]
```diff
@@ -389,7 +389,9 @@ const Bylaws = () => {
         <p>
           {' '}
           A vacancy in any office may be filled for the unexpired portion of the
-          term by a majority vote of the Board at any meeting.
+          term by a majority vote of the Board at any meeting. Each member of the
+          Board will cast one vote per vacancy. In case of tie, the President
+          will have a casting vote.
```
Comment (dzenanz) [inline src/pages/bylaws.js:394]: deciding instead of casting?

Comment (jhlegarreta) [reply src/pages/bylaws.js:394]: Had looked up the exact term: https://en.wikipedia.org/wiki/Casting_vote

File: src/pages/bylaws.js [hunk 2]
```diff
@@ -389,7 +389,9 @@ const Bylaws = () => {
         <p>
           {' '}
           A vacancy in any office may be filled for the unexpired portion of the
-          term by a majority vote of the Board at any meeting.
+          term by a majority vote of the Board at any meeting. Each member of the
```
Comment (thewtex) [inline src/pages/bylaws.js:392]: As discussed at the ISC meeting, this looks good to the Council, changing "majority" -> "plurality".

Comment (jhlegarreta) [reply src/pages/bylaws.js]: Done in 58861ae.

**Review by dzenanz — APPROVED** [review]

**Pull request merged**